src/assets/wise5/classroomMonitor/dataExport/strategies/LabelComponentDataExportStrategy.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '../../../common/Component';
+import { ComponentDataExportParams } from '../ComponentDataExportParams';
+import { ComponentRevisionCounter } from '../ComponentRevisionCounter';
+import { AbstractComponentDataExportStrategy } from './AbstractComponentDataExportStrategy';
+
+export class LabelComponentDataExportStrategy extends AbstractComponentDataExportStrategy {
+  maxNumLabels: number;
```

Comment:
Initialize value here, instead of in the constructor?
```suggestion
  maxNumLabels: number = 0;
```